tools/scripts/rewrite-awx-operator-requirements.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import yaml
+import sys
+
+
+AWX_ENTRY = 'https://github.com/ansible/awx.git#/awx_collection/'
+
+
+def load_yaml_file(fname):
+    with open(fname, 'r') as file:
+        data = yaml.safe_load(file)
+    return data
+
+
+def write_yaml_file(fname, data):
+    with open(fname, 'w') as file:
+        yaml.dump(data, file)
+
+
+def replace_awx(data, path):
+    for entry in data['collections']:
+        if entry['name'] == AWX_ENTRY or entry['name'] == 'awx.awx':
+            entry['name'] = f'file://{path}#/awx_collection/'
```

Comment:
is this to get the directory of the current branch?

Reply:
Effectively, yes.

This points at the current checkout of `awx` in the CI BUT because it is a git reference it will actually clone the current git repo @ the current branch. An alternative to this would be to use `type: dir` and save the extra clone.